tuning/config/configs.py

```diff
@@ -90,6 +90,14 @@ class TrainingArguments(transformers.TrainingArguments):
             'steps' (logging is done every `logging_steps`)"
         },
     )
+    trackers: Optional[List[str.lower]] = field(
+        default=None,
```

Comment:
the default here is None, but docstring says `file_logger(default),`

Reply:
fixed.